tests/frontier/precompiles/test_bn254.py

```diff
@@ -0,0 +1,62 @@
+"""Tests ecadd/ecmul/ecpairing precompiled contracts."""
+
+import pytest
+from execution_testing import (
+    Account,
+    Alloc,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from execution_testing.base_types.base_types import Address
+from execution_testing.forks import Byzantium, Istanbul
+from execution_testing.forks.helpers import Fork
+from execution_testing.vm import Opcodes as Op
+
+
+@pytest.mark.valid_from("Byzantium")
+@pytest.mark.parametrize(
+    "address,gas_pre_istanbul,gas_post_istanbul",
+    [
+        pytest.param(0x06, 500, 150, id="ecadd"),
+        pytest.param(0x07, 40_000, 6000, id="ecmul"),
+        pytest.param(0x08, 100_000, 45_000, id="ecpairing"),
```

Comment:
I'm not a fan of these hard-coded gas prices here, I think we can do better and put them inside the gas_prices object returned by each fork.